frontend/src/components/print/print-react/DownloadReactPdfListItem.vue

```diff
@@ -2,7 +2,7 @@
   <v-list-item :disabled="loading" @click="generatePdf">
     <v-list-item-icon>
       <v-icon v-if="loading" class="mdi-spin">mdi-loading</v-icon>
-      <v-icon v-else>mdi-react</v-icon>
+      <v-icon v-else>mdi-printer</v-icon>
     </v-list-item-icon>
     <v-list-item-title>
       {{ $tc('components.print.printReact.downloadNuxtPdfListItem.label') }}
```

Comment:
components.print.printReact.download _Nuxt_ PdfListItem.label
rename to
components.print.printReact.downloadReactPdfListItem.label

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/3036/commits/abe5bd69b9ce833ec7f3432f27a82c851004e91e